tests/infra/ccf.py

```diff
@@ -72,7 +72,6 @@ class Network:
         "gov_script",
         "join_timer",
         "worker_threads",
-        "domain",
```

Comment:
This reverts #1229 - is that deliberate?

Reply:
Old master, will rebase.